.travis.yml

```diff
@@ -6,9 +6,9 @@ virtualenv:
   system_site_packages: true
 
 env:
-  - VERSION="7.0" ODOO_REPO="odoo/odoo"
-  - VERSION="7.0" ODOO_REPO="OCA/OCB"
-  - VERSION="7.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
+  - VERSION="7.0" LINT_CHECK="1"
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" UNIT_TEST="1"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" UNIT_TEST="1"
```

Comment:
Integrated tests are missing. This should be:

    - VERSION="7.0" LINT_CHECK="1"
    - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
    - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
    - VERSION="7.0" UNIT_TEST="1" LINT_CHECK="0"